docs/integrations/sanic.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Sanic
+---
+
+# Sanic
+
+Strawberry comes with a basic Sanic integration. It provides a view that you can
+use to serve your GraphQL schema:
+
+```python
+from strawberry.sanic.views import GraphQLView
+
+from api.schema import Schema
+
+app = Sanic(__name__)
+
+app.add_route(
+    GraphQLView.as_view(schema=schema, graphiql=True),
+    "/graphql",
+)
+```
+
+## Options
+
+The `GraphQLView` accepts two options at the moment:
+
+- schema: mandatory, the schema created by `strawberry.Schema`.
+- graphiql: optional, defaults to `True`, whether to enable the GraphiQL
+  interface.
+
+## Extending the view
+
+We allow to extend the base `GraphQLView`, by overriding the following methods:
+
+- `get_context(self) -> Any`
+- `get_root_value(self) -> Any`
+- `process_result(self, result: ExecutionResult) -> GraphQLHTTPResponse`
+
+## get_context
+
+`get_context` allows to provide a custom context object that can be used in your
+resolver. You can return anything here, by default we return a dictionary with
+the request.
+
+```python
+class MyGraphQLView(GraphQLView):
+    def get_context(self, request) -> Any:
+        return {"example": 1}
+
+
+@strawberry.type
+class Query:
+    @strawberry.field
+    def example(self, info: Info) -> str:
+        return str(info.context["example"])
+```
+
+Here we are returning a custom context dictionary that contains only one item
+called "example".
+
+Then we use the context in a resolver, the resolver will return "1" in this
+case.
+
+## get_root_value
+
+`get_root_value` allows to provide a custom root value for your schema, this is
+probably not used a lot but it might be useful in certain situations.
```

Comment:
```suggestion
By overriding `GraphQLView.get_root_value` you can provide a custom root value for your
schema. This is probably not used a lot but it might be useful in certain situations.
```